src/components/board.ts

```diff
@@ -1,2 +1,2 @@
-export type { Board } from './board/board';
+export { type Board, type DurationOptions, PowerMode } from './board/board';
```

Comment:
[minor] see comment related to naming

```suggestion
export { type Board, type Duration, PowerMode } from './board/board';
```